src/reference/build.gradle

```diff
@@ -0,0 +1,72 @@
+buildscript {
+  dependencies {
+    classpath 'org.asciidoctor:asciidoctorj-epub3:1.5.0-alpha.6'
+    classpath 'org.asciidoctor:asciidoctorj-pdf:1.5.0-alpha.15'
```

Comment:
Aren't there some really GA versions for these plugins?

Reply:
Not that I can find. I experimented with a number of versions. This the "magic" combination that works. Asciidoctor-epub is especially fragile and requires some serious hoop-jumping (mostly in the attributes) to get to work.

Reply:
That sounds creepy. Are we really going to spend a lot of time coming up with the solution when dependencies we rely on are fragile ?

BTW I don't see that you have removed Dookbook artifacts, e.g. `index-docinfo.xml`.

Am I missing anything again?

Reply:
I forgot to remove that. Other than the Docbook-specific parts of the build, that file is the only Docbook-specific artifact. I'll zap it.